camel/retrievers/vector_retriever.py

```diff
@@ -72,7 +72,7 @@ def __init__(
 
     def process(
         self,
-        content: Union[str, Element],
+        content: Union[str, Element, IO[bytes]],
         chunk_type: str = "chunk_by_title",
         max_characters: int = 500,
```

Comment:
for the chunk part, add one more argument to let user decide whether to chunk the content